packages/contracts/contracts/minter-suite/Minters/MinterSetPricePolyptychERC20V5.sol

```diff
@@ -398,21 +393,24 @@ contract MinterSetPricePolyptychERC20V5 is
      * owned by msg.sender being used as the parent token.
      * @param ownedNFTTokenId ERC-721 NFT token ID owned by msg.sender to be
      * used as the parent token.
+     * @param purchasePricePerToken Maximum price of token being allowed by the purchaser, no decimal places
      * @return tokenId Token ID of minted token
      */
     function purchase(
         uint256 projectId,
         address coreContract,
         address ownedNFTAddress,
-        uint256 ownedNFTTokenId
+        uint256 ownedNFTTokenId,
+        uint256 purchasePricePerToken
```

Comment:
nit: very minor nit, but I think a little more straightforward to include the `maxPricePerToken` immediately after the `coreContract` arg here. That allows the ordering of args to be `[Fixed Price Minter Args],[Holder Args`].

Reply:
Good call! Updated 🙏